### ISSUE TITLE
Email notifications?

### ISSUE THREAD

**Issue by planetahuevo** [stripe/stripe-webhook-monitor#9]
Is it possible to get email notifications for the webhooks?

Thanks!

**Comment by willock-stripe** [comment]
Hi @planetahuevo –

Although [webhook events](https://stripe.com/docs/webhooks) are sometimes referred to as notifications, they are intended to notify _software_ of an event, rather than a human (where email would be more appropriate). For example, a webhook event from Stripe can inform your server that a direct debit payment has been successfully processed, so you could fulfil an order on their behalf.

If you want to be notified of certain events happening on your account (such as a payment being received), you can configure your [account email notifications](https://support.stripe.com/questions/set-up-account-email-notifications).

**Comment by planetahuevo** [comment]
Hi Willock. 
I understand perfectly what a webhook is, thank you for the explanation anyway. 
I do not want me to be notified but to setup an automated email for the things that are missing on Stripe, which are a lot, if you take a look into that link that you pasted above.
So it will be a machine sending the emails, because Stripe is missing some important ones, for example: payment failed notification to the stripe account.

**Comment by willock-stripe** [comment]
@planetahuevo – no worries.

Webhooks are only delivered to a user-provided HTTP endpoint presently.

A few options you could consider:

- The [Stripe Dashboard iOS app](https://apps.apple.com/gb/app/stripe-dashboard/id978516833) can provide details on successful and failed payments on your account, although I'm not certain the notifications can be customised to fit your use-case exactly
- Using a service provider that will parse your Webhook events and forward them on via email. Since this would be a third-party provider, there could be security and data privacy concerns. From some surface-level googling, a service like Automate.io or IFTTT could do this (this is absolutely not a recommendation for either service).
- Customise the code in this repository to send an email using a gateway such as Mailchimp (again, not a recommendation)

I will pass on the feedback about adding further email notifications – are there any in particular you feel are most important?

**Comment by planetahuevo** [comment]
Again, I know how webhooks work. Please stop doing that.
I have consider a lot of options and I am just asking if you have thought about integrating that feature (notifications) on your development. 
I know that there are other options out there and trust me, there is none working properly for this. I have been searching for that for a while, that is how I found your project, which looks great.
About your 3 options above, 1.- Not interesting on apps. 2.- I am looking for that provider to be your project, so it does not make sense. 3- read next line.

Also I am not a developer (but I know how webhooks work) so I cannot provide you with code to do this. I can help you debugging, commenting, giving you ideas and things like that. 

So, let me rephrase the whole topic, now that the answer to my question is "no".

Will you consider to add Notifications as a extra feature to your project? Email would be the best one, but could be also slack too. 

Thank you for your help.
good luck with the project. 

best

**Comment by planetahuevo** [comment]
About this: 
I will pass on the feedback about adding further email notifications – are there any in particular you feel are most important?
All the webhooks that do not have an email on your system, for example: failed payments to the owner of the account.
